Sources/PortalSwift/Mocks/Core/MockMpc/MockPortalMpc.swift

```diff
@@ -20,14 +20,13 @@ public class MockPortalMpc: PortalMpc {
     )
   }
 
-  override public func eject(
-    _: BackupMethods,
-    withCipherText _: String,
-    andOrganizationBackupShare _: String,
-    usingProgressCallback _: ((MpcStatus) -> Void)? = nil
-  ) async throws -> String {
-    return MockConstants.mockEip155EjectedPrivateKey
-  }
+    override public func eject(
+        _ method: BackupMethods,
+        with cipherText: String,
+        and organizationBackupShare: [PortalCurve : String]
+    ) async throws -> EjectedKeys {
+        return EjectedKeys(secp256k1Key: MockConstants.mockEip155EjectedPrivateKey, ed25519Key: MockConstants.mockSolonaEjectedPrivateKey)
+    }
```

Comment:
Indentation a bit off ^

Reply:
Yes yes, since I fromatted this func with Xcode using 'control + i', I believe the whole file needs to be formatted but I didn't want to include none-relevant changes to the PR.